docs/getting-started.md

```diff
@@ -54,6 +54,9 @@ Luigi consists of two main parts:
 * [General settings](general-settings.md) - other settings that can be configured in Luigi, such as HTML attributes, third-party cookie configuration and more.
 * [API](luigi-core-api.md) - functions to help with almost every part of your app: navigation, authorization, adding a [search box](luigi-core-api.md#globalsearch), configuring a light/dark [theme](luigi-core-api.md#theming) and others.
 
+<!-- add-attribute:class:success -->
+>**NOTE:** As of Luigi v2.5, you can use [Luigi Container](luigi-container.md) to embed a Luigi micro frontend into any existing application without the need of it being a Luigi Core app. 
```

Comment:
Luigi container is detached from core/client versioning and follows it's own release cycle